processors/config/anaSvtRawHitsMC_cfg.py

```diff
@@ -0,0 +1,41 @@
+import HpstrConf
+import sys
+import os
+
+# Use the input file to set the output file name
+infile = sys.argv[1].strip()
```

Comment:
Same comment as above, please use -i and -o options to pass in the input and output filenames from the command line.